package.json

```diff
@@ -12,10 +12,14 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-redux": "^7.1.7",
+    "bootstrap": "5.0.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.6.1",
```

Comment:
Do you really need this package in this project?

Reply:
some next commits use bootstrap components. In last commits I removed these components and then removed this package